lib/finishm/orfs_finder.rb

```diff
@@ -31,6 +34,15 @@ def validate_options(options, argv)
       visualise.validate_assembly_options(options)
   end
```

Comment:
I was thinking that it could use default names and output both if the user doesn't specify at least one of them?